internal/advertisement-operator/controller.go

```diff
@@ -18,8 +18,8 @@ package advertisement_operator
 import (
 	"context"
 	goerrors "errors"
-	discoveryv1alpha1 "github.com/liqoTech/liqo/api/discovery/v1alpha1"
 	configv1alpha1 "github.com/liqoTech/liqo/api/config/v1alpha1"
+	discoveryv1alpha1 "github.com/liqoTech/liqo/api/discovery/v1alpha1"
```

Comment:
why ci linter did tell it to me when I did this change?

Reply:
I think when you merged your PR that import was "policyv1" and this occurred when we modified the ClusterConfig version